### PR TITLE
Initial implementation for creating and getting racecars

### PR DESCRIPTION
**Pull request by sebinfrancis**

### CODE REVIEW

File: src/test/java/com/galvanize/speedway/racecars/RaceCarsIT.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+import org.springframework.test.web.servlet.MockMvc;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@AutoConfigureRestDocs
```
Comment (davidgrieser) [inline src/test/java/com/galvanize/speedway/racecars/RaceCarsIT.java:23]: I see the annotation but no `andDo` sections after expectations.

**Review by davidgrieser — DISMISSED** [review]
No REST docs added, but not sure if that will be done in a following PR. Simple consistent tests look good.

Make sure you don't forget to replace the `List` in the controller with actual database backing.

**Review by Christopher-Diehl-cognizant-com — APPROVED** [review]

File: src/main/java/com/galvanize/speedway/racecars/controller/RaceCarsController.java [hunk 2]
```diff
@@ -1,28 +1,27 @@
...
-    List<RaceCarDto> raceCars;
+    @Autowired
+    RaceCarsService raceCarsService;
 
-    RaceCarsController() {
-        raceCars = new ArrayList<>();
-    }
     @GetMapping
-    public List<RaceCarDto> getRaceCars() { return raceCars; }
+    public List<RaceCarDto> getRaceCars() { return raceCarsService.getAllRaceCars(); }
```
Comment (davidgrieser) [inline src/main/java/com/galvanize/speedway/racecars/controller/RaceCarsController.java:19]: Great change.

**Review by davidgrieser — APPROVED** [review]
Nice driving out of implementation.

**Comment by sebinfrancis** [comment]
Thanks

**Pull request merged**